src/views/ConflictResolution/index.tsx

```diff
@@ -1,151 +1,449 @@
+                                    oursValue,
+                                    theirsValue,
+
+                                    oursChanged,
+                                    theirsChanged,
+
+                                    oursSelected,
+                                    theirsSelected,
+
+                                    selected,
+                                    conflicted,
+                                }) => (
+                                    <Row
+                                        key={title}
+                                        leftClassName={styles.tagContainer}
+                                        left={(
+                                            <Tag
+                                                title={title}
+                                                value={originalValue}
+                                                disabled
+                                            />
+                                        )}
+                                        centerClassName={styles.tagContainer}
+                                        center={
+                                            ours && (
+                                                <Tag
+                                                    title={title}
+                                                    value={oursValue}
+                                                    changed={oursChanged}
+                                                    conflicted={!selected && conflicted}
+                                                    selected={oursSelected}
+                                                    disabled={!conflicted}
+                                                    onClick={() => this.handleTagClick(title, 'ours')}
+                                                />
+                                            )
+                                        }
+                                        rightClassName={styles.tagContainer}
+                                        right={
+                                            theirs && (
+                                                <Tag
+                                                    title={title}
+                                                    value={theirsValue}
+                                                    changed={theirsChanged}
+                                                    conflicted={!selected && conflicted}
+                                                    selected={theirsSelected}
+                                                    disabled={!conflicted}
+                                                    onClick={() => this.handleTagClick(title, 'theirs')}
+                                                />
+                                            )
+                                        }
+                                    />
+                                ))
```

Comment:
Looks way too complicated and unreadable.
Please separate it into chunks.
Remove the use of map and no inline functions.